src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlConnectionHelper.cs

```diff
@@ -21,6 +22,8 @@ public sealed partial class SqlConnection : DbConnection
         private DbConnectionOptions _userConnectionOptions;
         private DbConnectionPoolGroup _poolGroup;
         private DbConnectionInternal _innerConnection;
+        private bool _isExperimental;
```

Comment:
How is this set to true? Please also add code comment to describe why this property exists.